integration/projects_resources_get_test.go

```diff
@@ -206,14 +206,14 @@ IP             Region    Droplet ID    Droplet Name
       "sizes": [ "s-1vcpu-1gb" ],
       "features": [ "metadata" ],
       "available": true
-    },
```

Comment:
Looks like you might need to run `gofmt` on this, or maybe GitHub is rendering this diff weird.

Reply:
Or wait, this is a JSON string, `gofmt` won't help. I think this just needs to tweaked so the whitespace is consistent.

Reply:
Yea, that's actually part of a multiline string.  Looks like I had indentation set to use tabs.